tfexec/internal/testutil/tfcache.go

```diff
@@ -9,12 +9,13 @@ import (
 )
 
 const (
-	Latest011 = "0.11.15"
-	Latest012 = "0.12.31"
-	Latest013 = "0.13.7"
-	Latest014 = "0.14.11"
-	Latest015 = "0.15.5"
-	Latest_v1 = "1.0.0"
+	Latest011         = "0.11.15"
+	Latest012         = "0.12.31"
+	Latest013         = "0.13.7"
+	Latest014         = "0.14.11"
+	Latest015         = "0.15.5"
+	Latest_v1         = "1.0.0"
+	Latest_v1_1_alpha = "1.1.0-alpha20210811"
```

Comment:
```suggestion
	Latest_v1_1 = "1.1.0-alpha20210811"
```

Reply:
Technically this is an internal constant anyway, so it probably doesn't matter if we later rename it but I expect that we'd more likely be testing against any latest v1.1 once a stable version is out rather than keep testing against alpha versions.

Reply:
This makes sense, thank you!